R/zzz.R

```diff
@@ -0,0 +1,22 @@
+.onLoad <- function(libname, pkgname) {
+  # adapted from https://r-pkgs.org/code.html#sec-code-onLoad-onAttach
+  op <- options()
+  default_global_knitr <- list(teal.reporter.global_knitr = list(
+    echo = TRUE,
+    tidy.opts = list(width.cutoff = 60),
+    tidy = FALSE
+  ))
+
+  toset <- !(names(default_global_knitr) %in% names(op))
+  if (any(toset)) options(default_global_knitr[toset])
```

Comment:
```suggestion
  if (toset) options(default_global_knitr)
```
@kartikeyakirar there is only one element in `default_global_knitr` list

Reply:
If there is only one element, a clearer condition would be
`if ("teal.reporter.global_knitr" %in% names(options))`